src/SFA.DAS.LevyTransferMatching.Infrastructure/QueueNames.cs

```diff
@@ -8,5 +8,6 @@ public static class QueueNames
         public const string ApplicationApprovedEvent = "SFA.DAS.LevyTransferMatching.ApplicationApproved";
         public const string PledgeDebitFailed = "SFA.DAS.LevyTransferMatching.PledgeDebitFailed";
         public const string TransferRequestApprovedEvent = "SFA.DAS.LTM.TransferRequestApproved";
+        public const string ApplicationFundingDeclined = "SFA.DAS.LevyTransferMatching.ApplicationFundingDeclined";

```

Comment:
There's a hidden 50-character limit to queue names - could you change LevyTransferMatching to LTM - thanks

Reply:
Ah, that would cause issues. Changed to `"SFA.DAS.LTM.ApplicationFundingDeclined"`.